geoh5py/objects/points.py

```diff
@@ -128,35 +119,54 @@ def remove_vertices(
             raise ValueError("Found indices larger than the number of vertices.")
 
         vertices = np.delete(self.vertices, indices, axis=0)
```

Comment:
you can remove the isinstance self.vertices

Reply:
True